frontend/src/Routes.js

```diff
@@ -4,13 +4,15 @@ import AlertConfiguration from './AlertConfiguration/AlertConfigurationComponent
 import AlertsManagement from './AlertsManagement/AlertsManagement';
 import App from './App';
 import ErrorPage from './ErrorPage';
+import History from './AlertsManagement/History';
 
 /** Create routes for application. */
 export default function Routes() {
   return (
     <Switch>
       <Route exact path="/" component={App} />
       <Route path="/alerts" component={AlertsManagement} />
+      <Route path="/history" component={History} />
       <Route path="/configs" component={AlertConfiguration} />
       <Route component={ErrorPage} />
```

Comment:
yup it will

Reply:
WAIT update I just realized that it won't.... because if the user tries to access an invalid id, it will throw an Error right now instead of redirecting to the Error Page. I'll fix this in the error handling PR tho.

Reply:
okay, thanks for double checking!
because i think it errored out once when i was playing around lol